diff --git a/node_modules/rn-phone-input-field/dist/main/constants/constants.js b/node_modules/rn-phone-input-field/dist/main/constants/constants.js
index b79c930..7a7fea3 100644
--- a/node_modules/rn-phone-input-field/dist/main/constants/constants.js
+++ b/node_modules/rn-phone-input-field/dist/main/constants/constants.js
@@ -1266,7 +1266,7 @@ var constants = {
         countryCode: 'GB',
         countryName: 'United Kingdom',
         callingCode: '44',
-        regex: '^(\\+44|44)?[1-9]\\d{8}$',
+        regex:`^(\\+44|44|0)7\\d{9}$`,
     },
     US: {
         icon: '🇺🇸',
diff --git a/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js b/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
index 436e38e..a221c65 100644
--- a/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
+++ b/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
@@ -75,6 +75,7 @@ var PhoneInput = (0, react_1.forwardRef)(function (_a, ref) {
             if (!(text === null || text === void 0 ? void 0 : text.length))
                 return false;
             var fullNumber = country.callingCode + text;
+
             return validationRegex.test(fullNumber);
         },
         defaultCountry: function (code) {
